### ISSUE TITLE
Staging Review finding: Link opens in a new tab without warning

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128349]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments Team
**Product:** Appointments
**Feature:** Oracle Health Scheduling - Direct Schedule and Request for an Appointment

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @jasonday (Accessibility)

### Description

On the "which type of care?" page, the link in the alert to Find your local VA health care facility opens in a new tab (as it should!) but doesn't indicate that it will open in a new tab.

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="729" height="179" alt="Image" src="https://github.com/user-attachments/assets/08b27781-157e-4275-8756-5061c3ac5c5e" />

</details>

## Recommended action

Use the [link - external](https://design.va.gov/components/link#external) variant for this link, which will append `(opens in a new tab)` to the link text.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Link component](https://design.va.gov/components/link)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
